runtime/compiler/env/VMJ9.cpp

```diff
@@ -7560,7 +7553,7 @@ TR_J9VM::inlineNativeCall(TR::Compilation * comp, TR::TreeTop * callNodeTreeTop,
                break;
 
             // Skip methods with java.lang.invoke.FrameIteratorSkip annotation
-            if ((J9_ROM_METHOD_FROM_RAM_METHOD(callerMethod)->modifiers & J9AccMethodFrameIteratorSkip) != J9AccMethodFrameIteratorSkip)
+            if (TR::Compiler->mtd.isFrameIteratorSkipMethod(callerMethod))
```

Comment:
This is wrong, should be:
```suggestion
            if (!TR::Compiler->mtd.isFrameIteratorSkipMethod(callerMethod))
```